### PR TITLE
Develop menuitem

### PR DESCRIPTION
**Pull request by irmakyuceak**

### CODE REVIEW

**Review by uguryildiz — CHANGES_REQUESTED** [review]
Menu Item width must be same size to other items. However sub-menu width can be more than root menu items. You can review for sample on profile widget.

**Comment by irmakyuceak** [comment]
item size changed

File: src/widgets/views/notifications.php [hunk 1]
```diff
@@ -49,7 +49,7 @@
     <ul class="nav">
         <li class="nav-item dropdown">
             <a class="nav-link dropdown-toggle" href="#" role="button" id="notificationDropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                <i class="fa fa-bell-slash" style="padding-left: 10px; padding-right: 20px; margin-top: 3px; color: black;"></i>
+                <i class="fa fa-bell-slash" style="padding-left: 5px; padding-right: 5px; margin-top: 3px; color: black;"></i>
             </a>
             <div class="dropdown-menu" aria-labelledby="notificationDropdown">
                 <h6 class="dropdown-header">No Notifications</h6>
```
Comment (uguryildiz) [inline src/widgets/views/notifications.php:55]: "No Notifications" header must change as "Notifications"

**Review by uguryildiz — CHANGES_REQUESTED** [review]
Please check messages.

**Review by uguryildiz — APPROVED** [review]

**Pull request merged**